src/report.py

```diff
@@ -10,26 +10,25 @@ class PortDict(TypedDict, total=False):
     change: float
 
 
+def to_portdict(data: Dict[str, Any]) -> PortDict:
```

Comment:
Could you also add the related exercise(s) to the function or module docstring so that we can check these? I think @martinp7 you asked for this here? https://github.com/l-mizrahi/Practical-Python-Exercises/pull/1#discussion_r678984471

Reply:
Okay, this is fine, we can work it out through the blame. I just struggled to find which exercise this function originated from